### PR TITLE
Lots of improvements

### PR DESCRIPTION
**Pull request by mccalluc**
@afred, please review. At this point I know that I need to add options for configuring machine and volume type/size, and I'd like to add high-level tests of the whole workflow.

### CODE REVIEW

File: lib/core/base_wrapper.rb [hunk 1]
```diff
@@ -8,9 +8,7 @@
 
 module BaseWrapper
   
-  Aws.config[:region] = 'us-east-1' # One-time configuration at module load.
-  
-  LOGGER = Logger.new($stdout, 'daily')
+  LOGGER = Logger.new($stderr, 'daily')
```
Comment (afred) [inline lib/core/base_wrapper.rb]: why this change?

Comment (mccalluc) [inline lib/core/base_wrapper.rb]: @afred: The ssh-args script is intended to be run in backticks, so there we want to be sure that only the "real" output goes to STDOUT.

File: lib/core/elb_wrapper.rb [hunk 2]
```diff
@@ -11,8 +11,13 @@ def elb_client
   
   public
   
-  def elb_arn(region, account_id, elb_name)
-    "arn:aws:elasticloadbalancing:#{region}:#{account_id}:loadbalancer/#{elb_name}"
+  def elb_arn(elb_name)
+    account_id = Aws::IAM::CurrentUser.new.arn.match(/^arn:aws:iam::(\d+)/)[1]
+    "arn:aws:elasticloadbalancing:#{availability_zone}:#{account_id}:loadbalancer/#{elb_name}"
+  end
```
Comment (afred) [inline lib/core/elb_wrapper.rb]: @mccalluc this seems like an odd way of getting the ELB's ARN -- although I can't seem to find it anywhere in the AWS web console either. Where did you find the it?

Aside, I tried using `Aws::ElasticLoadBalancer.new.describe_load_balancers` and it's unable to authenticate... so not sure what's going on there. I've set the credentials in `Aws.config` to the values in my `~/.aws/credentials` file, but no dice.

Comment (mccalluc) [inline lib/core/elb_wrapper.rb]: (Talked in person: The user ARN has the account ID, which we use for building another ARN. There could be a better way.)

File: lib/script_helper.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require 'yaml'
+
+module ScriptHelper
+  def self.read_config(binding)
+    YAML.load_file(__dir__+'/../scripts/config.yml').each do |key, value|
+      binding.local_variable_set(key.to_sym, value)
+    end
+  end
+end
```
Comment (afred) [inline lib/script_helper.rb]: @mccalluc if i understand this correctly, it loads all values from `scripts/config.yml` as local variables wherever `ScriptHelper.read_config(binding)` is called?

If that's the case i'm wondering what is the value of having config values as locally defined variables in the current binding, as opposed to just doing something like `config["config_key"]`)? Defining local variables outside of the local scope seems odd. Also, mightn't you accidentally overwrite a variable?

Comment (afred) [inline lib/script_helper.rb]: i also notice that there is no `scripts/config.yml`

Comment (mccalluc) [inline lib/script_helper.rb]: @afred: Good calls on both points. 
- On the `binding` thing, I had started off with the local variables, and when I needed to add the default machinery I was curious if code outside the current scope could tweak the binding... but just because you can, doesn't mean it's a good idea.
- On the other point, not secrets per se, but details that will differ between accounts. Also not necessarily required, everything could be supplied on the command line, but more clear if there was a message.
Thanks.
